goovn/ovndb.go

```diff
@@ -75,31 +73,28 @@ type OVNDB struct {
 	imp *ovnDBImp
 }
 
-var once sync.Once
-var ovnDBApi OVNDBApi
```

Comment:
why we remove that?

Reply:
we must keep only one instance in one process.

Reply:
Ah, ok.. I'm prefer to use client like semantics..
Does it strict requirement to have single instance?